src/main/java/htsjdk/tribble/AbstractFeatureReader.java

```diff
@@ -237,16 +236,42 @@ public Object getHeader() {
     }
 
     public static boolean isTabix(String resourcePath, String indexPath) throws IOException {
-        if(indexPath == null){
+        try (SeekableStream inputStream = SeekableStreamFactory.getInstance().getStreamFor(resourcePath)) {
```

Comment:
if this throws IOException it's better to just propagate it rather than suppress it to false I think.